src/rhsm/connection.py

```diff
@@ -1257,15 +1291,15 @@ def hypervisorCheckIn(self, owner, env, host_guest_mapping, options=None):
 
             query_params = urlencode(params)
             url = "/hypervisors/%s?%s" % (owner, query_params)
-            res = self.conn.request_post(url, host_guest_mapping)
+            res = self.conn.request_post(url, host_guest_mapping, description=_("Registering hypervisor"))
```

Comment:
I think this might be more descriptive as "Updating detected virtual machines running on this host"

Reply:
It needn't to be this host. It can be any host. So I would say that the message should be: "Updating detected virtual machines running on given host". Thoughts?

Reply:
Yea that works. I'm really not sure the best phrasing but I think that gets the point across.